### ISSUE TITLE
Output:  Add a token picker

### ISSUE THREAD

**Issue by Sidnioulz** [Sidnioulz/design-tokens-docs#5]
## Example

https://atlassian.design/components/tokens/all-tokens

![Image](https://github.com/user-attachments/assets/f60e267b-9300-4c47-8d00-7cece3944f31)

## Impact

Designers often struggle to adopt a design system's tokens and to find the right token for a usage. They would be greatly helped if they had a tool that lets them describe their need and that makes proposals.

This is in line with the Adobe DS team's concept of [codifying design data](https://www.youtube.com/watch?v=KmIh96ABSiE), i.e. addressing tokens by context of use rather than name.

## Challenges

There is no standard or de facto standard for describing token taxonomies and semantics. While some orgs have developed tailored pickers, a generic tool cannot be built without a shared understanding of the meanings given to various elements of a token taxonomy.

**Comment by drwpow** [comment]
> This is in line with the Adobe DS team's concept of [codifying design data](https://www.youtube.com/watch?v=KmIh96ABSiE), i.e. addressing tokens by context of use rather than name.

I really like that train of thought—being able to search for tokens not only by names, but usages. I could see it just as useful to search for components and descriptions, and have those tokens appear in the search results

**Comment by Sidnioulz** [comment]
Yeah, if this could be codified, it could make for a great advanced search tool. My issue is: I don't know how generalisable this is. Can we qualify the role of each taxonomy element in a DT taxonomy, so that the right questions can be asked by a generic picker?

It feels like an entire project to me.
